.travis.yml

```diff
@@ -17,7 +17,7 @@ cache:
     - ~/.cache
 
 git:
-  depth: 50
+  depth: 3
```

Comment:
We can reset the git clone depth back to 3 since we're no longer looking at the history to determine which modules need to be published.